src/analog/commands/DataSegmentBase.ts

```diff
@@ -50,4 +52,13 @@ export default class DataSegmentBase extends Command {
 
         return new this(buffer.getDataSegment());
     }
+
+    toJson ( conversionFormatId: number = HEX, conversionFormatOptions: TBytesFormatOptions = {} ) {
```

Comment:
maybe move this method to parent Command?

Reply:
The parent class Command already has.
`toJson () {`
`    return JSON.stringify(this.getParameters());`
`}`